src/sentry/ingest/types.py

```diff
@@ -6,10 +6,16 @@ class ConsumerType:
     Events = "events"  # consumes simple events ( from the Events topic)
     Attachments = "attachments"  # consumes events with attachments ( from the Attachments topic)
     Transactions = "transactions"  # consumes transaction events ( from the Transactions topic)
+    Occurrences = "occurrences"  # consumes issue occurrences ( from the Occurrences topic)
 
     @staticmethod
     def all():
-        return (ConsumerType.Events, ConsumerType.Attachments, ConsumerType.Transactions)
+        return (
+            ConsumerType.Events,
+            ConsumerType.Attachments,
+            ConsumerType.Transactions,
+            # ConsumerType.Occurrences,
```

Comment:
Not totally sure that we need to include this here, this will allow this consumer to be run as an ingest consumer, which is defined as https://github.com/getsentry/sentry/blob/8624bde15ccc84dfb6b5b82f7e69c20b746e45bd/src/sentry/runner/commands/run.py#L537-L538

This consumer accepts occurrences so it might make sense to treat it separately.

Reply:
Will we want it to be an ingest consumer eventually? Or fully separate?

Reply:
Yeah a bit tricky to figure out where this stuff might live...ideally it would be in here with this and then just be able to spin up its own consumer separately but the run commands just like to run one consumer per command

Reply:
I think we'd probably treat this as fully separate. If we find out we're wrong we could always add it back here though